### ISSUE TITLE
Add badge API/tools for making READMEs look better

### ISSUE THREAD

**Issue by M4cs** [M4cs/Pretty-Readme#9]
For this issue feel free to add a link to any badge tools you think are useful.

**Comment by SuvamPrasd** [comment]
what kind of badges do you want to include? @M4cs

**Comment by M4cs** [comment]
We are looking for links to repos/APIs that offer badges for READMEs. See the badges section of the repo at the moment.

**Comment by SuvamPrasd** [comment]
Great, so you want to add more relevant badges to the readme file that should be useful to everybody ?

**Comment by M4cs** [comment]
Yes if they are a resource based around badges or they are a service that offers badges. The badges at the top of the repository should be good and kept the way they are.

**Comment by SuvamPrasd** [comment]
Okay, great.. I will go for this enhancement issue.
